airbyte-integrations/connectors/source-confluence/metadata.yaml

```diff
@@ -25,5 +25,4 @@ data:
   ab_internal:
     sl: 200
     ql: 300
-  supportLevel: community
```

Comment:
This key is duplicated and the other entry's value is `certified`. Which one is correct? I did this change assuming that `certified` overrides `community`.

Reply:
Note that in [the connector report](https://connectors.airbyte.com/files/generated_reports/connector_registry_report.html) the Confluence source shows up as `community`.